config/default/manager_auth_proxy_patch.yaml

```diff
@@ -3,23 +3,23 @@
 apiVersion: apps/v1
```

Comment:
Please fix indentation in this file and I think that's it

Reply:
Corrected